blips/blip-0001.md

```diff
@@ -0,0 +1,289 @@
+general, we'd like to retain the original author as a co-author of the transferred bLIP, 
+but that's really up to the original author. A good reason to transfer ownership is 
+because the original author no longer has the time or interest in updating it or 
+following through with the bLIP process, or has fallen off the face of the 'net (i.e. is 
+unreachable or not responding to email). A bad reason to transfer ownership is because 
+you don't agree with the direction of the bLIP. We try to build consensus around a bLIP, 
+but if that's not possible, you can always submit a competing bLIP.
+
+If you are interested in assuming ownership of a bLIP, send a message asking to take over, 
+addressed to both the original author and the bLIP editor. If the original author doesn't 
+respond to email in a timely manner, the bLIP editor will make a unilateral decision (it's 
+not like such decisions can't be reversed).
+
+### bLIP Editors
+
+The current BOLT editors are the bLIP editors.
+
+### bLIP Editor Responsibilities & Workflow
+
+For each new bLIP submission, the editors do the following:
+
+* Read the bLIP to check if it is ready: sound and complete. The ideas must make technical 
+sense, even if they don't seem likely to get to final status.
+* The title should accurately describe the content.
+* The bLIP draft must have been sent to the lightning-dev mailing list for discussion.
+* Motivation and backward compatibility (when applicable) must be addressed.
+* Licensing terms must be acceptable for bLIPs.
+
+If the bLIP isn't ready, the editor will send it back to the author for revision, with 
+specific instructions.
+
+Once the bLIP is ready for the repository, the bLIP editor will:
+
+* Assign a bLIP number (generally the PR number or, if preferred by the author, the Issue # 
+if there was discussion in the Issues section of this repository about this bLIP)
+* Check the requested Feature Bit, Message Type, and/or TLV assignments for collisions.
+* Merge the corresponding pull request
+* Send a message back to the bLIP author with the next steps.
+
+The bLIP editors are intended to fulfill administrative and editorial responsibilities. 
+They do not pass judgement on bLIPs. The bLIP editors monitor bLIP changes, and update bLIP 
+headers as appropriate.
+
+## What belongs in a successful bLIP?
+
+bLIPs should be written in Markdown format.
+
+Each bLIP should have the following parts:
+
+* **Preamble** -- Headers containing metadata about the bLIP (see below).
+* **Abstract** -- A short (~200 word) description of the technical issue being addressed.
+* **Copyright** -- The bLIP must be explicitly licensed under acceptable copyright terms (see below).
+* **Specification** -- The technical specification should describe the syntax and semantics 
+of any new feature. The specification should be detailed enough to allow competing, 
+interoperable implementations for any of the current Lightning implementations.
```

Comment:
nit: shouldn't this be included *after* the motivation and rationale?

Reply:
fixed and appreciated!